### DISCUSSION TITLE
I win a block and the  reward do not arrive

### DISCUSSION THREAD

**Discussion by jealalri** [Chia-Network/chia-blockchain#8542] [Support]
I win a block in space pool but the reward  no show in my wallet

**Comment by loppefaaret** [comment]
perhaps if you tell me what block number and what address you got in your "Farming rewards target address", i might be able to verify your supposedly win ?

**Comment by jealalri** [comment]
It is no supposedly win, I already verify that, they say chia blockchain sent directly the 0.25 xch to my farmer reward adress


Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jealalri** [comment]
The block number is 887426

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by loppefaaret** [reply]
block 887426 was won by poolharvest with their own farmer - not by a farmer on pool.space
could you provide the address that was credited 0.25 please ? that way i can quickly track down the block

**Reply by loppefaaret** [reply]
and please verify, that the address that was credited 0.25, is also the address listed in your "Farmer Rewards Target Address" under the 3 `...` menu on the "Farm" tab ?

**Comment by loppefaaret** [comment]
<sup>Originally posted by **jealalri** September 22, 2021</sup>
I won a block in space pool, i got this 
Fecha
9/20/21, 10:47 PM
Altura
887426
Hash
1af00ac572a09975433d8bec790998390e93814646fd13c6d1937fad0046c4b9
My farmer Id is: 
0xd86a7d4ff00ca5545c375093756812d094815db40529738fc9f68c6bf8912eb5
the reward wallet where they are suppose to send the token is: xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p
and what you do is close the case</div>

**Comment by loppefaaret** [comment]
so... pool.space - i have a few users say they been told by them, that they have a block win - and it turned out that it wasnt so when looking back onto the blockchain data - that being said, now that we got the block number, and the hashes, and address, lets see what we can see:

**Reply by loppefaaret** [reply]
one thing i need from you, could you goto the "Farm" tab in the Chia app, and under the 3 `...` on the top right, open up the box with your rewards target addresses, and tell me what you got in the "Farmer rewards target address" ? 
with that i am able to track down any block win your farmer have had while having that address set!

**Reply by loppefaaret** [reply]
block number 887426 was indeed rewarding xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p - but i doubt you actually control this address to be honest - it is a pool controlled farmer, tied into a pool address from what i can see, belonging to poolharvest.io
https://xchscan.com/blocks/887426
https://www.chiaexplorer.com/blockchain/address/xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p

**Comment by jealalri** [comment]
xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p  this the address, but I already change it

________________________________
De: loppefaaret ***@***.***>
Enviado: miércoles, 22 de septiembre de 2021 10:25 a. m.
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: jealalri ***@***.***>; Author ***@***.***>
Asunto: Re: [Chia-Network/chia-blockchain] I win a block and the reward do not arrive (Discussion #8542)


one thing i need from you, could you goto the "Farm" tab in the Chia app, and under the 3 ... on the top right, open up the box with your rewards target addresses, and tell me what you got inb the "Farmer rewards target address" ?
with that i am able to track down any block win your farmer have had while having that address set!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Chia-Network/chia-blockchain/discussions/8542#discussioncomment-1369689>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVRXL5R73VANRBPS224SOT3UDHRPDANCNFSM5EN4O43Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by jealalri** [comment]
And then they steal the token

________________________________
De: loppefaaret ***@***.***>
Enviado: miércoles, 22 de septiembre de 2021 10:28 a. m.
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: jealalri ***@***.***>; Author ***@***.***>
Asunto: Re: [Chia-Network/chia-blockchain] I win a block and the reward do not arrive (Discussion #8542)


block number 887426 was indeed rewarding xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p - but i doubt you actually control this address to be honest - it is a pool controlled farmer, tied into a pool address from what i can see, belonging to poolharvest.io
https://xchscan.com/blocks/887426
https://www.chiaexplorer.com/blockchain/address/xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Chia-Network/chia-blockchain/discussions/8542#discussioncomment-1369701>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVRXL5VYSQ2BUUVNRV75U3DUDHRZPANCNFSM5EN4O43Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by plot14** [reply]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?

**Comment by loppefaaret** [comment]
are you a owner / operator of the poolharvest.io pool ? that address belongs to that pool - when you got that address - `xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p` in you "Farmer rewards target address" - does it show a warning message above, saying that you dont have a key that controls the address ?

**Comment by jealalri** [comment]
yes it has tha warning, and I open a new gui, and get a new address

________________________________
De: loppefaaret ***@***.***>
Enviado: miércoles, 22 de septiembre de 2021 10:38 a. m.
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: jealalri ***@***.***>; Author ***@***.***>
Asunto: Re: [Chia-Network/chia-blockchain] I win a block and the reward do not arrive (Discussion #8542)


are you a owner / operator of the poolharvest.io pool ? that address belongs to that pool - when you got that address - xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p in you "Farmer rewards target address" - does it show a warning message above, saying that you dont have a key that controls the address ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Chia-Network/chia-blockchain/discussions/8542#discussioncomment-1369756>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVRXL5XD5E5L5HNXDLPV7QDUDHS5JANCNFSM5EN4O43Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by loppefaaret** [reply]
that means it is not an address coming from your key - you dont control that address
are you able to tell me what address that was there before you started changing it ? 
if not, you could try and run the CLI command `chia keys show` and tell me the "First wallet address" of your key - this is the default address chia will set as your "Farming rewards target address"

**Comment by jealalri** [comment]
xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p
this addres was in the reward target address, with the warnig
and this is the address where I get 0.01 XCh every day xch148wk9qcc24p7jk6s8yhlpkv58plvhgzerjwx3uty6a5hxlufaqjsqcppes

________________________________
De: loppefaaret ***@***.***>
Enviado: miércoles, 22 de septiembre de 2021 10:51 a. m.
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: jealalri ***@***.***>; Author ***@***.***>
Asunto: Re: [Chia-Network/chia-blockchain] I win a block and the reward do not arrive (Discussion #8542)


that means it is not an address coming from your key - you dont control that address
are you able to tell me what address that was there before you started changing it ?
if not, you could try and run the CLI command chia keys show and tell me the "First wallet address" of your key - this is the default address chia will set as your "Farming rewards target address"

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Chia-Network/chia-blockchain/discussions/8542#discussioncomment-1369866>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVRXL5USXJIEHL7VGTAI7Y3UDHUNJANCNFSM5EN4O43Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by loppefaaret** [reply]
ok, i cant explain how that address got in there, but it needs to be _any_ receive address from you own wallet - if you control `xch148wk9qcc24p7jk6s8yhlpkv58plvhgzerjwx3uty6a5hxlufaqjsqcppes` - that is one option of an address you could use there
the other 0.25 that went to `xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p` are lost

**Comment by jealalri** [comment]
ok, thanks

________________________________
De: loppefaaret ***@***.***>
Enviado: miércoles, 22 de septiembre de 2021 11:03 a. m.
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: jealalri ***@***.***>; Author ***@***.***>
Asunto: Re: [Chia-Network/chia-blockchain] I win a block and the reward do not arrive (Discussion #8542)


ok, i cant explain how that address got in there, but it needs to be any receive address from you own wallet - if you control xch148wk9qcc24p7jk6s8yhlpkv58plvhgzerjwx3uty6a5hxlufaqjsqcppes - that is one option of an address you could use there
the other 0.25 that went to xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p are lost

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Chia-Network/chia-blockchain/discussions/8542#discussioncomment-1369932>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVRXL5SZL6VYPTHZU53QSELUDHV4BANCNFSM5EN4O43Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by AleksandrPDA** [reply]
Hello. Did you manage to solve your problem?

**Reply by loppefaaret** [reply]
> Hello. Did you manage to solve your problem?

if you got a similar issue, perhaps try and explain your situation, the addresses involved, in a new support discussion

**Reply by AleksandrPDA** [reply]
yes, I have the same problem, the program shows that I won the block, but 0.25 did not reach my wallet

**Reply by plot14** [reply]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?

**Comment by shooti22** [comment]
Good Morning,
i have the same issuie, but i was mining solo and 1.5 chia did Not arrive
LG M.

**Comment by jealalri** [comment]
No

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by plot14** [comment]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?
